tests/test_html_output.py

```diff
@@ -57,6 +60,23 @@ def browser():
     firefox.close()
 
 
+def test_cli_log_custom_name(tmpdir):
+    log_name = "custom_name"
```

Comment:
You are mutating the global variable inside test.
What if the execution order of tests changes?

Reply:
There is no `global log_name` line at the start of the test, so the `log_name` global variable will be not changed.
But I've updated the variable name to clarify that it's local.

Reply:
Ah, I was confused by the variable name. Thank you for the fix.